rtgui/cropwindow.cc

```diff
@@ -272,210 +269,209 @@ void CropWindow::buttonPress (int button, int type, int bstate, int x, int y)
 {
 
     bool needRedraw = true;  // most common case ; not redrawing are exceptions
+    EditSubscriber *editSubscriber = iarea->getCurrSubscriber();
```

Comment:
This code has been done before you jumped into the project. I'll change to `auto` here but won't spend time to hide all pointer type introduced (where it applies in this patch). I'll try to think about doing it for future ones.